apps/omg_watcher/test/web/controllers/challenge_test.exs

```diff
@@ -51,6 +52,7 @@ defmodule OMG.Watcher.Web.Controller.ChallengeTest do
     } = TestHelper.success?("utxo.get_challenge_data", %{"utxo_pos" => utxo_pos})
   end
 
+  @tag skip: true
```

Comment:
you should probably recreate this test (to be precise - test the same behavior) in the "new approach tests".

Ah this requires you to tweak the `Challenger.create_challenge` a bit, along the lines of:
```
with spending_blknum_response <- OMG.DB.spent_blknum({blknum, txindex, oindex}),
     {:ok, spending_blknum} <- Core.ensure_is_challengable(spending_blknum_response), # other possible return here is {:error, :some_appropriate_error_ideally_one_that_we_used_to_throw}
....
```

Reply:
I added test to [challenger_core](https://github.com/omisego/elixir-omg/blob/79146aaa846b62d5eb88dc53e5f316aed3526116/apps/omg_watcher/test/challenger/core_test.exs#L91)